server/fleet/activities.go

```diff
@@ -49,6 +49,10 @@ var ActivityDetailsList = []ActivityDetails{
 	ActivityTypeEditedMacOSMinVersion{},
 
 	ActivityTypeReadHostDiskEncryptionKey{},
+
+	ActivityTypeCreatedMacosProfile{},
+	ActivityTypeDeletedMacosProfile{},
```

Comment:
@gillespi314 I added the created and deleted activities while I was at it, don't know if that's something you already took care of in your API endpoints PR? If so I can just remove them from here.

Reply:
No, this is perfect. Thank you so much!